fat/fat2/pticker.go

```diff
@@ -0,0 +1,111 @@
+package fat2
+
+import "fmt"
+
+// PPTicker is an internal representation of a PegNet asset type
+type PTicker int
+
+const (
+	PTickerInvalid PTicker = iota
+	PTickerPEG
+	PTickerUSD
+	PTickerEUR
+	PTickerJPY
+	PTickerGBP
+	PTickerCAD
+	PTickerCHF
+	PTickerINR
+	PTickerSGD
+	PTickerCNY
+	PTickerHKD
+	PTickerKRW
+	PTickerBRL
+	PTickerPHP
+	PTickerMXN
+	PTickerXAU
+	PTickerXAG
+	PTickerXBT
+	PTickerETH
+	PTickerLTC
+	PTickerRVN
+	PTickerXBC
+	PTickerFCT
+	PTickerBNB
+	PTickerXLM
+	PTickerADA
+	PTickerXMR
+	PTickerDAS
+	PTickerZEC
+	PTickerDCR
+	PTickerMax
+)
+
+// TODO: figure out how to handle "tUSD" and other testnet types
+//       or convince PegNet core devs that the differentiation is unnecessary
```

Comment:
I support making it the same. Don't change the damn thing we're trying to test to be cute. you can quote me on that.

Reply:
Totally agree. I'll resolve this and keep the TODO for now. We can bring up the topic to the pegnet core-dev channel